resip/dum/DialogSet.cxx

```diff
@@ -635,6 +635,19 @@ DialogSet::dispatch(const SipMessage& msg)
             }
             break;
 
+         case OPTIONS:
```

Comment:
You could reduce code duplication here, by only breaking if dialog is not null, thus falling through to the default case if dialog is null.